### PR TITLE
Documentation Review

### PR DESCRIPTION
**Pull request by aalborov**
Still in progress, couldn't find some models

### CODE REVIEW

**Comment by vladimir-dudnik** [comment]
why it is in WIP state then?

**Comment by aalborov** [comment]
@vladimir-dudnik fixed

File: models/intel/human-pose-estimation-0001/description/human-pose-estimation-0001.md [hunk 1]
```diff
@@ -35,7 +37,7 @@ Tested on a COCO validation subset from the original paper: Cao et al. ["Realtim
 
 ## Outputs
 
-1. The net outputs two blobs with shapes: [1, 38, 32, 57] and [1, 19, 32, 57]. The first blob contains keypoint pairwise relations (part affinity fields), the second one contains keypoint heatmaps.
+The net outputs two blobs with the [1, 38, 32, 57] and [1, 19, 32, 57] shapes. The first blob contains keypoint-pairwise relations (part affinity fields), while the second blob contains keypoint heatmaps.
```
Comment (IRDonch) [inline models/intel/human-pose-estimation-0001/description/human-pose-estimation-0001.md]: > keypoint-pairwise relations

I don't think that's correct. They are not relations that are "keypoint-pairwise", they are pairwise relations of keypoints.

File: models/intel/person-reidentification-retail-0031/description/person-reidentification-retail-0031.md [hunk 2]
```diff
@@ -22,13 +22,13 @@ This is a person reidentification model for a general scenario. It uses a whole
 | Source framework                  | Caffe*                                    |
 
 The cumulative matching curve (CMC) at rank-1 is accuracy denoting the possibility to locate at least one true positive in the top-1 rank.
-Mean Average Precision (mAP) is the mean across all queries’ Average Precision (AP) and AP is defined as an area under the [precision/recall](https://en.wikipedia.org/wiki/Precision_and_recall) curve.
+Mean Average Precision (mAP) is the mean across all queries Average Precision (AP). AP is defined as the area under the [precision and recall](https://en.wikipedia.org/wiki/Precision_and_recall) curve.
```
Comment (IRDonch) [inline models/intel/person-reidentification-retail-0031/description/person-reidentification-retail-0031.md]: Why did you remove the apostrophe here? It's supposed to be possessive.

File: models/intel/product-detection-0001/description/product-detection-0001.md [hunk 3]
```diff
@@ -2,8 +2,8 @@
 
 ## Use Case and High-Level Description
 
-Product detector based on SSD-lite architecture with [MobileNetV2](https://arxiv.org/pdf/1801.04381.pdf) as a backbone for Self-checkout Point of Sale related scenes.
-The detector is able to detect 12 classes of objects (`sprite`, `kool-aid`, `extra`, `ocelo`, `finish`, `mtn_dew`, `best_foods`, `gatorade`, `heinz`, `ruffles`, `pringles`, `del_monte`) and first labels 0 and 1 are related to `background_label` and `undefined` correspondingly.
+The product detector based on the SSD-lite architecture with [MobileNetV2](https://arxiv.org/pdf/1801.04381.pdf) as a backbone for self-checkout points of sale-related scenes.
+The detector can recognize 12 classes of objects (`sprite`, `kool-aid`, `extra`, `ocelo`, `finish`, `mtn_dew`, `best_foods`, `gatorade`, `heinz`, `ruffles`, `pringles`, `del_monte`). Labels 0 and 1 are related to `background_label` and `undefined` correspondingly.
```
Comment (IRDonch) [inline models/intel/product-detection-0001/description/product-detection-0001.md]: > The detector can recognize 12 classes of objects

This is a detector, so it detects, not recognizes. I think we should avoid mixing terminology.

File: models/intel/product-detection-0001/description/product-detection-0001.md [hunk 4]
```diff
@@ -2,8 +2,8 @@
 
 ## Use Case and High-Level Description
 
-Product detector based on SSD-lite architecture with [MobileNetV2](https://arxiv.org/pdf/1801.04381.pdf) as a backbone for Self-checkout Point of Sale related scenes.
-The detector is able to detect 12 classes of objects (`sprite`, `kool-aid`, `extra`, `ocelo`, `finish`, `mtn_dew`, `best_foods`, `gatorade`, `heinz`, `ruffles`, `pringles`, `del_monte`) and first labels 0 and 1 are related to `background_label` and `undefined` correspondingly.
+The product detector based on the SSD-lite architecture with [MobileNetV2](https://arxiv.org/pdf/1801.04381.pdf) as a backbone for self-checkout points of sale-related scenes.
```
Comment (IRDonch) [inline models/intel/product-detection-0001/description/product-detection-0001.md]:
```suggestion
A product detector based on the SSD-lite architecture with [MobileNetV2](https://arxiv.org/pdf/1801.04381.pdf) as a backbone for self-checkout points of sale-related scenes.
```

This would be more logical and consistent with other models.

File: models/intel/text-spotting-0001-detector/description/text-spotting-0001-detector.md [hunk 5]
```diff
@@ -1,15 +1,15 @@
 # text-spotting-0001-detector
 
-## Use case and High-level description
+## Use Case and High-Level Description
 
-This is text spotting model that means it simultaneously detects and
-recognizes text. The model detects symbol sequences separated by space and does
-recognition without using any dictionary. The model is built on top of Mask-RCNN
+This is a text-spotting model that simultaneously detects and
```
Comment (IRDonch) [inline models/intel/text-spotting-0001-detector/description/text-spotting-0001-detector.md]: > text-spotting

This is analogous to "text detection" or "text recognition", so I don't think there should be a hyphen there.

File: models/intel/text-spotting-0001-detector/description/text-spotting-0001-detector.md [hunk 6]
```diff
@@ -19,37 +19,36 @@ This model is Mask-RCNN-based text detector with ResNet50 backbone and additiona
...
 
 ## Inputs
 
-1.	name: `im_data` , shape: [1x3x768x1280] - An input image in the format
-    [1xCxHxW]. The expected channel order is BGR.
-1.	name: `im_info`, shape: [1x3] - Image information: processed image height,
-    processed image width and processed image scale
+1.	Name: `im_data` , shape: [1x3x768x1280]. An input image in the [1xCxHxW] format. 
+    The expected channel order is BGR.
+2.	Name: `im_info`, shape: [1x3]. An input image with a processed image height,
```
Comment (IRDonch) [inline models/intel/text-spotting-0001-detector/description/text-spotting-0001-detector.md]: This isn't an image. It is _information_ about the input image, as the original description suggests.

File: models/public/efficientnet-b0-pytorch/efficientnet-b0-pytorch.md [hunk 7]
```diff
@@ -2,15 +2,19 @@
 
 ## Use Case and High-Level Description
 
-The `efficientnet-b0-pytorch` model is one of the [EfficientNet](https://arxiv.org/abs/1905.11946) group of models designed to perform image classification. This model was pretrained in PyTorch\*. All the EfficientNet models have been pretrained on the ImageNet image database. For details about this family of models, check out the [repository](https://github.com/rwightman/gen-efficientnet-pytorch).
+The `efficientnet-b0-pytorch` model is one of the [EfficientNet](https://arxiv.org/abs/1905.11946) models designed to perform image classification. This model was pretrained in PyTorch\*. All the EfficientNet models have been pretrained on the ImageNet\* image database. For details about this family of models, check out the [TensorFlow EfficientNet repository](https://github.com/rwightman/gen-efficientnet-pytorch).
```
Comment (IRDonch) [inline models/public/efficientnet-b0-pytorch/efficientnet-b0-pytorch.md]: This link isn't to the TensorFlow EfficientNet repository.

File: models/public/efficientnet-b0/efficientnet-b0.md [hunk 8]
```diff
@@ -2,11 +2,11 @@
...
 
-The `efficientnet-b0` model is one of the [EfficientNet](https://arxiv.org/abs/1905.11946)
-group of models designed to perform image classification.
+The `efficientnet-b0` model is one of the [EfficientNet](https://arxiv.org/abs/1905.11946) models 
+designed to perform image classification.
 This model was pretrained in TensorFlow\*.
-All the EfficientNet models have been pretrained on the ImageNet image database.
-For details about this family of models, check out the [repository](https://github.com/tensorflow/tpu/tree/master/models/official/efficientnet).
+All the EfficientNet models have been pretrained on the ImageNet\* image database.
+For details about this family of models, check out the [TensorFlow EfficientNet repository](https://github.com/tensorflow/tpu/tree/master/models/official/efficientnet).
```
Comment (IRDonch) [inline models/public/efficientnet-b0/efficientnet-b0.md]: Strictly speaking, this isn't the _EfficientNet_ repository. It is the TensorFlow Cloud TPU repository (which is where the EfficientNet models happen to be).

File: models/public/efficientnet-b0-pytorch/efficientnet-b0-pytorch.md [hunk 9]
```diff
@@ -2,15 +2,19 @@
...
-before passing the image blob into the network. In addition, values must be divided
-by [58.395,57.12,57.375].
+### Input 
+
+The model input is a blob that consists of a single image with the [3x224x224] shape in the RGB
+order. Before passing the image blob to the network, do the following:
+1. Subtract the RGB mean values as follows: [123.675,116.28,103.53]
+2. Divide the RGB mean values by  [58.395,57.12,57.375]
+
+### Output
```
Comment (eizamaliev) [inline models/public/efficientnet-b0-pytorch/efficientnet-b0-pytorch.md]: Please, leave this section in `Use Case and High-Level Description`, because it part of description

File: models/intel/text-spotting-0001-detector/description/text-spotting-0001-detector.md [hunk 10]
```diff
@@ -19,37 +19,36 @@ This model is Mask-RCNN-based text detector with ResNet50 backbone and additiona
...
-1.	name: `boxes`, shape: [100x4] - Bounding boxes around every detected objects
-    in (top_left_x, top_left_y, bottom_right_x, bottom_right_y) format.
-1.	name: `raw_masks`, shape: [100x2x28x28] - Segmentation heatmaps for all
+1.	Name: `boxes`, shape: [100x4]. Bounding boxes around every detected object
+    in the (top_left_x, top_left_y, bottom_right_x, bottom_right_y) format.
+1.	Name: `raw_masks`, shape: [100x2x28x28]. Segmentation heatmaps for all
     classes for every output bounding box.
-1.  name: `text_features`, shape [100x64x28x28] - Text features that are fed to
-text recognition head.
+1.  Name: `text_features`, shape [100x64x28x28]. Text features fed to a text recognition head.
```
Comment (eizamaliev) [inline models/intel/text-spotting-0001-detector/description/text-spotting-0001-detector.md]: Original text is logically correct, output features are input for another type model

File: models/intel/text-spotting-0001-detector/description/text-spotting-0001-detector.md [hunk 11]
```diff
@@ -19,37 +19,36 @@ This model is Mask-RCNN-based text detector with ResNet50 backbone and additiona
...
 
-1.	name: `im_data` , shape: [1x3x768x1280] - An input image in the format
-    [1xCxHxW]. The expected channel order is BGR.
-1.	name: `im_info`, shape: [1x3] - Image information: processed image height,
-    processed image width and processed image scale
+1.	Name: `im_data` , shape: [1x3x768x1280]. An input image in the [1xCxHxW] format. 
+    The expected channel order is BGR.
+2.	Name: `im_info`, shape: [1x3]. Image information: processed image height,
+    processed image width and processed image scale.
     w.r.t. the original image resolution.
```
Comment (aalborov) [inline models/intel/text-spotting-0001-detector/description/text-spotting-0001-detector.md]: @eizamaliev what does this line relate to?

File: models/public/efficientnet-b0-pytorch/efficientnet-b0-pytorch.md (hunk 9 above)
Comment (aalborov) [reply models/public/efficientnet-b0-pytorch/efficientnet-b0-pytorch.md]: It is a children element of  `Use Case and High-Level Description`, just like the `Input` section.

**Comment by aalborov** [comment]
@IRDonch  thanks for the comments!

File: models/intel/person-reidentification-retail-0031/description/person-reidentification-retail-0031.md (hunk 2 above)
Comment (IRDonch) [reply models/intel/person-reidentification-retail-0031/description/person-reidentification-retail-0031.md]: Please replicate the edit to the other `person-reidentification-retail-*` networks, as well.

File: models/intel/text-spotting-0001-detector/description/text-spotting-0001-detector.md (hunk 5 above)
Comment (IRDonch) [reply models/intel/text-spotting-0001-detector/description/text-spotting-0001-detector.md]: Please fix other instances of this too.

File: models/public/efficientnet-b0-pytorch/efficientnet-b0-pytorch.md (hunk 7 above)
Comment (IRDonch) [reply models/public/efficientnet-b0-pytorch/efficientnet-b0-pytorch.md]: Please update the other links to this URL as well.

File: models/public/efficientnet-b0/efficientnet-b0.md (hunk 8 above)
Comment (IRDonch) [reply models/public/efficientnet-b0/efficientnet-b0.md]: Please update the other links to this URL as well.

File: models/public/brain-tumor-segmentation-0002/brain-tumor-segmentation-0002.md [hunk 12]
```diff
@@ -13,37 +13,47 @@ This model was created for participation in the [Brain Tumor Segmentation Challe
...
+    - edema 
+    - enhancing tumor 
+- TC (*tumor core*) combines the following classes:
+    - necrotic core / non-enhancing tumor
+    - enhancing tumor
+
+The following accuracy metrics are measured on a `brain tumor` training subset of the [Medical Decathlon](http://medicaldecathlon.com/) dataset. 
+
+**Mean**:  
+Dice index for "overall": 0.915
```
Comment (IRDonch) [inline models/public/brain-tumor-segmentation-0002/brain-tumor-segmentation-0002.md]: Please strip the trailing spaces.

File: models/public/brain-tumor-segmentation-0002/brain-tumor-segmentation-0002.md [hunk 13]
```diff
@@ -67,55 +77,32 @@ The channels are ordered as `T1`, `T1ce`, `T2`, `Flair`.
...
+[MIT License](https://raw.githubusercontent.com/lachinov/brats2019/master/LICENSE).
 
-THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
-THE SOFTWARE.
-```
```
Comment (IRDonch) [inline models/public/brain-tumor-segmentation-0002/brain-tumor-segmentation-0002.md]: What's the motivation for this change? This is how the licenses are documented for all public models.

File: models/public/human-pose-estimation-3d-0001/description/human-pose-estimation-3d-0001.md [hunk 14]
```diff
@@ -21,19 +21,19 @@ Multi-person 3D human pose estimation model based on [Lightweight OpenPose](http
...
     - H - image height
     - W - image width
 
-   Expected color order - BGR.
+   Expected color order is BGR.
 
 ## Outputs
 
-1. The net outputs three blobs with shapes: [1, 57, 32, 56], [1, 19, 32, 56], and [1, 38, 32, 56]. The first blob contains coordinates in 3D space, the second one contains keypoint heatmaps and the third is keypoint pairwise relations (part affinity fields).
+The net outputs three blobs with shapes the following shapes: `[1, 57, 32, 56]`, `[1, 19, 32, 56]`, and `[1, 38, 32, 56]`. The first blob contains coordinates in 3D space, the second blob contains keypoint heatmaps and the third blob is keypoint pairwise relations (part affinity fields).
```
Comment (IRDonch) [inline models/public/human-pose-estimation-3d-0001/description/human-pose-estimation-3d-0001.md]:
```suggestion
The net outputs three blobs with the following shapes: `[1, 57, 32, 56]`, `[1, 19, 32, 56]`, and `[1, 38, 32, 56]`. The first blob contains coordinates in 3D space, the second blob contains keypoint heatmaps and the third blob is keypoint pairwise relations (part affinity fields).
```

File: models/public/efficientnet-b0-pytorch/efficientnet-b0-pytorch.md (hunk 9 above)
Comment (eizamaliev) [reply models/public/efficientnet-b0-pytorch/efficientnet-b0-pytorch.md]: Please, leave original structure of `Use Case...` section, because it can be used for internal usage.

File: models/public/brain-tumor-segmentation-0002/brain-tumor-segmentation-0002.md (hunk 13 above)
Comment (aalborov) [reply models/public/brain-tumor-segmentation-0002/brain-tumor-segmentation-0002.md]: Don't we usually provide the link only? The text here matches what's in the link.

Comment (IRDonch) [reply models/public/brain-tumor-segmentation-0002/brain-tumor-segmentation-0002.md]: For each model, there is a link to the license and a copy of that license. In the cases where the license is short, it is embedded directly in the README for easy reference.

Comment (aalborov) [reply models/public/brain-tumor-segmentation-0002/brain-tumor-segmentation-0002.md]: Okay I'll fix that

File: models/public/efficientnet-b0-pytorch/efficientnet-b0-pytorch.md (hunk 9 above)
Comment (aalborov) [reply models/public/efficientnet-b0-pytorch/efficientnet-b0-pytorch.md]: okay, fixed

File: models/intel/text-spotting-0001-detector/description/text-spotting-0001-detector.md (hunk 11 above)
Comment (aalborov) [reply models/intel/text-spotting-0001-detector/description/text-spotting-0001-detector.md]: @eizamaliev  Please assist here

Comment (IRDonch) [reply models/intel/text-spotting-0001-detector/description/text-spotting-0001-detector.md]: I can tell you. It's a continuation of the sentence on the previous line. You put a period at the end there, which you shouldn't have.

Comment (aalborov) [reply models/intel/text-spotting-0001-detector/description/text-spotting-0001-detector.md]: Right, thanks

**Comment by IRDonch** [comment]
Jenkins: test this please

File: models/public/brain-tumor-segmentation-0002/brain-tumor-segmentation-0002.md [hunk 15]
```diff
@@ -13,37 +13,48 @@ This model was created for participation in the [Brain Tumor Segmentation Challe
...
+See [BRATS 2019 Leaderboard](https://www.cbica.upenn.edu/BraTS19/lboardValidation.html). The metrics 
+for challenge validation (Dice_WT, Dice_TC, Dice_ET) differ from the metrics reported below (which 
+are compartible with input labels):
+- WT (*whole tumor*) class combines all three tumor classes:
+    - necrotic core / non-enhancing tumor
+    - edema 
+    - enhancing tumor 
+- TC (*tumor core*) combines the following classes:
+    - necrotic core / non-enhancing tumor
+    - enhancing tumor
```
Comment (eizamaliev) [inline models/public/brain-tumor-segmentation-0002/brain-tumor-segmentation-0002.md]: TC stands for necrotic core and non-enhancing tumor, and ET (or DICE_ET) - for enhancing tumor. So correct structure will be something like this:

>- WT (*whole tumor*) class combines all three tumor classes:
>    - necrotic core / non-enhancing tumor
>    - edema 
>    - enhancing tumor 
>- TC (*tumor core*) combines the following classes:
>    - necrotic core 
>    - non-enhancing tumor
> - ET - enhancing tumor

Comment (alinaalborova) [reply models/public/brain-tumor-segmentation-0002/brain-tumor-segmentation-0002.md]: Fixed

**Comment by IRDonch** [comment]
Jenkins: test this please
